src/app/resources/resources.component.html

```diff
@@ -60,6 +60,10 @@ <h3 class="header"><a *ngIf="!nationName" [routerLink]="['/resources/view', elem
             <mat-icon>delete_forever</mat-icon>
             <span i18n>Delete</span>
           </button>
+          <a href={{resourceSrc}} mat-menu-item (click)="downloadFile(element)" download [disabled]="!element.isDownloadable">
```

Comment:
Again, you don't need a click handler.  Just set the href to the url for the file to download.

This does, however, highlight the point that the filename property isn't set for our mock data.  I'll make a new issue for that.